scss/core/tokens/_sizing.scss

```diff
@@ -0,0 +1,54 @@
+// stylelint-disable declaration-no-important
+
+// Width and height
+
+@each $prop, $abbrev in (width: w, height: h) {
+  @each $size, $length in $sizes {
+    .#{$abbrev}-#{$size} { #{$prop}: $length !important; }
+  }
+}
+
+//Widths
+
+.mw-100 { max-width: 100% !important; }
+.mh-100 { max-height: 100% !important; }
+
+.w-auto{
+  width: auto;
+}
+
+@each $size, $value in $widths {
+  .w-#{$size} {
+    width: 1rem * $value !important;
+  }
+}
+
+// Maya icon sizes
+
+.dci {
+  vertical-align: middle;
+  font-size: 1em;
+}
```

Comment:
icons related class could go in icons related files

Reply:
Done.